js/packages/cli/src/helpers/transactions.ts

```diff
@@ -118,7 +118,8 @@ export async function sendSignedTransaction({
   } catch (err) {
     log.error('Timeout Error caught', err);
     if (err.timeout) {
-      throw new Error('Timed out awaiting confirmation on transaction');
+      log.error('timed out but continuing');
+      //throw new Error('Timed out awaiting confirmation on transaction, timeout is ' + DEFAULT_TIMEOUT);
```

Comment:
Not sure on what i think about this one..  but note that this file is used across other things where if a transaction failed it must exist and this change is braking behaviour